docs/build/migrations/02-upgrading.md

```diff
@@ -0,0 +1,850 @@
+# Upgrading Cosmos SDK
```

Comment:
confused on why this was part of gitignore?

cc @samricotta

Reply:
This should never have been committed there :/
This is copied from UPGRADING.md